processor.go

```diff
@@ -313,6 +313,9 @@ func (p *processor) send(clientIP net.Addr, reqID uuid.UUID, tenant string, wr *
 	req.Header.Set("X-Prometheus-Remote-Write-Version", "0.1.0")
 	req.Header.Set("X-Cortex-Tenant-Client", clientIP.String())
 	req.Header.Set("X-Cortex-Tenant-ReqID", reqID.String())
+	if p.cfg.Tenant.LabelPrefix != "" {
+		tenant = fmt.Sprintf("%s-%s", p.cfg.Tenant.LabelPrefix, tenant)
```

Comment:
I'm thinking maybe we should omit the `-` and let people choose themselves (maybe they want an underscore? :) )
So I'd just go with `tenant = p.cfg.Tenant.Prefix + tenant` (this should also be a bit faster than `fmt`)

Reply:
I would also provide a link or list the delimiters allowed for use in the README or comments to the config
https://grafana.com/docs/mimir/latest/configure/about-tenant-ids/

Reply:
Fixed